cristin-biobank/src/main/java/no/unit/nva/biobank/model/cristin/CristinBiobank.java

```diff
@@ -0,0 +1,189 @@
+package no.unit.nva.biobank.model.cristin;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.time.Instant;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+
+@SuppressWarnings("PMD.ExcessiveParameterList")
+public final class CristinBiobank {
+    //Here be more params
+    private static final String CRISTIN_BIOBANK_ID = "cristin_biobank_id";
+    private static final String CRISTIN_NAME_FIELD = "name";
+
+    private static final String CRISTIN_BIOBANK_COORDINATING_INSTITUTION = "institution";
+
+    private static final String CRISTIN_BIOBANK_LAST_MODIFIED = "last_modified";
+    private static final String CRISTIN_BIOBANK_ASSOC_PROJECT = "associated_project";
+    private static final String CRISTIN_BIOBANK_TYPE = "type";
+    private static final String CRISTIN_BIOBANK_MAIN_LANGUAGE_KEY = "main_language";
+    private static final String CRISTIN_BIOBANK_START_DATE = "start_date";
+    private static final String CRISTIN_BIOBANK_STORED_UNTIL_DATE = "stored_until_date";
+    private static final String CRISTIN_BIOBANK_STATUS = "status";
+    private static final String CRISTIN_BIOBANK_CREATED = "created";
+    private static final String CRISTIN_BIOBANK_COORDINATOR = "coordinator";
+    private static final String CRISTIN_BIOBANK_EXTERNAL_SOURCES = "external_sources";
+    private static final String CRISTIN_BIOBANK_APPROVALS = "approvals";
+    private static final String CRISTIN_BIOBANK_MATERIALS = "biobank_material";
+
+    @JsonProperty(CRISTIN_BIOBANK_ID)
+    private final String cristinBiobankId;
+
+    @JsonProperty(CRISTIN_BIOBANK_TYPE)
+    private final String cristinBiobankType;
+    @JsonProperty(CRISTIN_NAME_FIELD)
+    private final Map<String, String> name;
+    @JsonProperty(CRISTIN_BIOBANK_MAIN_LANGUAGE_KEY)
+    private final String cristinBiobankLanguage;
+
+    @JsonProperty(CRISTIN_BIOBANK_START_DATE)
+    private final Instant cristinBiobankStartDate;
```

Comment:
We already know that the field is a part of the class CristinBiobank, so I do not think it is necessary to prefix all field names in class CristinBiobank with cristinBiobank. Maybe i need a second opinion

Reply:
I did mostly for myself and future code exploration. We do know it, but when someone tries to navigate the code it makes it way easier to have it prefixed to avoid confusion?

Reply:
Well, class name is CristinBiobank. It is much easier to have a field named startDate(). All the extra chatter prefixing all fields with cristinBiobank is just noise in my opinion.